examples/demo-app/src/components/load-data-modal/load-remote-map.js

```diff
@@ -0,0 +1,155 @@
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const propTypes = {
+  onLoadRemoteMap: PropTypes.func.isRequired
+};
+
+const StyledDescription = styled.div`
+  font-size: 14px;
+  color: ${props => props.theme.labelColorLT};
+  line-height: 18px;
+  margin-bottom: 12px;
+`;
+
+const InputForm = styled.div`
+  flex-grow: 1;
+  padding: 32px;
+  background-color: ${props => props.theme.panelBackgroundLT};
+`;
+
+const StyledInput = styled.input`
+  width: 100%;
+  padding: ${props => props.theme.inputPadding};
+  color: ${props => props.error ? 'red' : props.theme.titleColorLT};
+  height: ${props => props.theme.inputBoxHeight};
+  border: 0;
+  outline: 0;
+  font-size: ${props => props.theme.inputFontSize};
+  
+  :active,
+  :focus,
+  &.focus,
+  &.active {
+    outline: 0;
+  }
+`;
+
+const StyledFromGroup = styled.div`
+  margin-top: 30px;
+  display: flex;
+  flex-direction: row;
+`;
+
+export const StyledInputLabel = styled.div`
+  font-size: 11px;
+  color: ${props => props.theme.textColorLT};
+  letter-spacing: 0.2px;
+  ul {
+    padding-left: 12px;
+  }
+`;
+
+export const StyledBtn = styled.button`
+  background-color: ${props => props.theme.primaryBtnActBgd};
+  color: ${props => props.theme.primaryBtnActColor};
+`;
+
+export const StyledError = styled.div`
+  color: red;
+`;
+
+export const StyledErrorDescription = styled.div`
+  font-size: 14px;
+`;
+
+const Error = ({error, url}) => (
+  <StyledError>
+    <StyledErrorDescription>{url}</StyledErrorDescription>
+    <StyledErrorDescription>{error.message}</StyledErrorDescription>
+  </StyledError>
+);
+
+class LoadRemoteMap extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      dataUrl: ''
+    };
+  }
+  onMapUrlChange = (e) => {
+    // TODO: validate url
+    this.setState({
+      dataUrl: e.target.value
+    });
+  };
+
+  onLoadRemoteMap  = () => {
+    const {dataUrl} = this.state;
+    if (!dataUrl) {
+      return;
+    }
+
+    this.props.onLoadRemoteMap({dataUrl});
+  };
+
+  render() {
+    return (
+      <div>
+        <InputForm>
+          <StyledDescription>Load your map using your custom URL</StyledDescription>
+          <StyledInputLabel>
+            You can use the following formats: CSV | JSON | KEPLERGL.JSON. Make sure the url contains the file extension.
```

Comment:
is KEPLERGL.JSON a required file extension to load the config json? if not, just mention, the json format can also be a kepler.gl config json. Otherwise it's a little confusing